### PR TITLE
First Attempt

### PR DESCRIPTION
**Pull request by adrianeyre**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,78 +14,342 @@ Airport Challenge
...
+
+  def initialize
+    rand(10) == 0 ? @sunny = false : @sunny = true
+  end
+
+  def sunny?
+    sunny
+  end
+end
+```
```
Comment (danldb) [inline README.md]: why is all your code in here?

File: lib/airplane.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class Airplane
+  attr_accessor :landed
```
Comment (danldb) [inline lib/airplane.rb]: never use public attr_accessors

File: lib/airport.rb [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    fail "Airplane is not at this airport"
+  end
+
+  private
+
+  def check_plane(airplane)
+    airplanes.each_with_index do |check,index|
+      next unless check == airplane
+      airplanes.delete_at(index)
+      airplane.landed = false
```
Comment (danldb) [inline lib/airport.rb]: directly changing another objects internal state is a very bad thing

File: README.md (hunk 1 above)
Comment (adrianeyre) [inline README.md]: Sorry, it was for my benefit and I thought it might be nice for future employees to see my work clearly.

**Pull request closed without merging**